public/app.css

```diff
@@ -102,3 +117,58 @@ html, body {
     float: right;
     box-sizing: border-box;
 }
+
+.groupsSection {
+    height: 100%;
+    width: 100%;
+    --search-width: 400px;
+    --details-width: calc(100% - var(--search-width) - 1px);
+}
+
+.groupSearch {
+    float: left;
+    box-sizing: border-box;
+    width: var(--search-width);
+    padding: 5px 20px 0px 5px;
+}
+
+.groupSearchForm {
+    width: 100%;
+    height: 25px;
+}
+
+.groupSearchFormInput {
+    box-sizing: border-box;
+    width: calc(100% - 50px);
+    height: 100%;
+}
+
+.groupSearchFormButton {
+    float: right;
+    box-sizing: border-box;
+    padding: 0px;
+    width: 50px;
+    height: 100%;
+}
+
+.groupDetails {
+    background-color: rgba(128, 64, 0, 0.1);
+    float: left;
+    box-sizing: border-box;
+    width: var(--details-width);
+    height: 100%;
+    padding-left: 10px;
+}
+
+.groupInfo {
+
+}
+
+.groupDescription {
+    width: calc(50px + 70%);
+    max-width: 100%;
+}
+
+.groupOptions {
+    padding-top: 15px;
+}
```

Comment:
In general, be a bit careful with css variables. It's actually a very new tech and not supported in all browsers (read: IE) scss/sass might be a better idea for the time being.